src/signals/incident-management/containers/Dashboard/components/BarChart/vega-config-bar-chart.ts

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+export const vegaConfigBarChart: any = {
```

Comment:
Why `any` as type? This should be a type of vega-lite

Reply:
Just as a placeholder until I figured out which type it was. Changed it to VisualizationSpec